tests/test_data/pinned.json

```diff
@@ -10,7 +10,7 @@
   "delegations": [
     {
       "paths": ["*"],
-      "repositories": ["mainrepo", "director"]
+      "repositories": ["imagerepo", "director"]
```

Comment:
Not a great one, no. Normally, if it's on its own, I just write 'imagerepo', and if it's part of some phrase in which words are already separated (like TEST_IMAGE_REPO_ROOT_FNAME), I separate the words.

There's an exception I can think of - there's an add_target_to_imagerepo call in the demo. That should probably be separated.